index.html

```diff
@@ -137,6 +143,25 @@ <h4>
           </dl>
         </section>
         <section>
+          <h4>
+            <dfn>canShareFiles()</dfn> method
+          </h4>
+          <p>
+            Each user agent should have an `allowlist` of permitted MIME types
+            and file extensions or file names. These may contain wildcards, for
+            example `*.txt` or `text/*`.
+          </p>
+          <p>
+            When the {{Navigator/canShareFiles()}} method is called with
+            argument |file_types|, run the following steps:
+          </p>
+          <ol class="algorithm">
+            <li>For each |file_type| in |file_types|, if |file_type| does not
+            match any entry in the user agent's `allowlist`, return `false`.
```

Comment:
For the mime types, they need to be [parsed](https://mimesniff.spec.whatwg.org/#parse-a-mime-type) (which can return "failure", so throw a TypeError), and then the "[essence](https://mimesniff.spec.whatwg.org/#mime-type-essence)" of each type can be checked against the allowlist.